boards/arm/pan1783_pan1783a_pan1783apa_evb/pan1783a_pa_evb_cpuapp.dts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023 Panasonic Industrial Devices Europe GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf5340_cpuapp_qkaa.dtsi>
+#include "pan1783_pan1783a_pan1783apa_cpuapp_common.dtsi"
+
+/ {
+	model = "Panasonic PAN1783A-PA EVB (NRF5340) Application";
+	compatible = "panasonic,pan1783a_pa-evb-cpuapp";
+
+	chosen {
+		zephyr,sram = &sram0_image;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,sram-secure-partition = &sram0_s;
+	};
+
+	nrf_radio_fem: fem_node {
+		compatible = "skyworks,sky66407-11",
+			     "generic-fem-two-ctrl-pins",
+			     "nordic,nrf-gpio-forwarder";
+		status = "okay";
+		ctx-gpios = <&gpio0 19 GPIO_ACTIVE_HIGH>;
+		crx-gpios = <&gpio0 21 GPIO_ACTIVE_HIGH>;
+	};
```

Comment:
This does not seem to be correct. I think you rather need something like:
```
	&gpio_fwd {
		status = "okay";
		fem {
			gpios = <&gpio0 19 0>, <&gpio0 22 0>;
		};
	};
```

Reply:
Ah okay, yeah because the only thing the appcore needs to know is that these two pins needs to be forwarded right?
I just did it this way because it was suggested in a Nordic Devzone Ticket that way:

> Is the overlay below already added to the application core? Can you try to modify and add it to child_image/remote_shell/board/nrf5340dk_nrf5340_cpuapp.overlay 
```
/ {
    nrf_radio_fem: fem_node {
        compatible = "skyworks,sky66407-11", "generic-fem-two-ctrl-pins";
        ctx-gpios = <&gpio0 19 GPIO_ACTIVE_HIGH>;
        crx-gpios = <&gpio0 21 GPIO_ACTIVE_HIGH>;
     };
};
```
from: https://devzone.nordicsemi.com/support/311726

But I will try your version and change it accordingly, if everything is still working. Thanks a lot!

Reply:
> Ah okay, yeah because the only thing the appcore needs to know is that these two pins needs to be forwarded right?

Yes.

> But I will try your version and change it accordingly, if everything is still working. Thanks a lot!

You'll need to also add `/delete-node/ uart;` to what I suggested if you don't want UART pins to be forwarded by default.

Reply:
Thanks for pointing me to this! Haven't noticed, that this will overwrite my disabled status in the `pan1783a_pa_evb_cpuapp.dts`. I will change that of course.